src/utils/pagination/getItemsOnPageCount.ts

```diff
@@ -0,0 +1,11 @@
+export const getItemsOnPageCount = (
+  currentPage: number,
+  perPage: number,
+  items: Array<unknown>,
+): number[] => {
+  const first =
+    currentPage === 1 ? currentPage : currentPage * perPage - perPage + 1;
```

Comment:
The calculation for `first` is mostly correct, but the condition `currentPage === 1` is unnecessary because the formula `currentPage * perPage - perPage + 1` already handles the case for the first page correctly. Consider simplifying the expression to `currentPage * perPage - perPage + 1` for all cases.